modules/installation-about-custom-aws-vpcs.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// * installing/
+
+[id="installation-about-custom-aws-vpcs_{context}"]
+= About using a custom VPC
+
+In {product-title} {product-version}, you can deploy a cluster into an existing VPC in Amazon Web Services (AWS). If you do, you must also use existing subnets within the VPC and routing rules.
+
+By deploying {product-title} into an existing AWS VPC, you might be able to avoid limit constraints in new accounts or more easily abide by the operational constraints that your company's guidelines set. This is a good option to use if you cannot obtain the infrastructure creation permissions that are required to create the VPC yourself.
+
+[id="installation-about-custom-aws-vpcs-requirements_{context}"]
+== Requirements for using your VPC
+
+The installation program will no longer create the following components:
```

Comment:
hmm maybe designing it in a way that describe that installer would be doing X to provide Y, but now the user needs to make sure Y is still possible by doing X or Z as per their choice...

usually the installer creates `Internet gateways`, `NAT gateways`, `Route tables` for subnets to make sure the cluster has connectivity to the Internet.

In bring your VPC case, the user needs to make sure the subnets for the cluster are similarly configured to provide connectivity to the Internet.

Reply:
Do you have an up-to-date description of the preferred configuration?

Reply:
[The UPI VPC template][1] is the preferred configuration.

[1]: https://docs.openshift.com/container-platform/4.2/installing/installing_aws_user_infra/installing-aws-user-infra.html#installation-cloudformation-vpc_installing-aws-user-infra